ResearchKit/ActiveTasks/ORKActiveStepViewController.m

```diff
@@ -362,6 +363,9 @@ - (void)finish {
     if (self.activeStep.shouldVibrateOnFinish) {
         AudioServicesPlayAlertSound(kSystemSoundID_Vibrate);
     }
+    if (self.activeStep.hasVoice && self.activeStep.finishedSpokenInstruction) {
+        [[ORKVoiceEngine sharedVoiceEngine] speakText:self.activeStep.finishedSpokenInstruction];
```

Comment:
Is the `finishedSpokenInstruction` overlapped with next step's `spokenInstruction`?